packages/common/src/index.ts

```diff
@@ -44,6 +44,7 @@ export * from "d3-drag";
 export * from "d3-dsv";
 export * from "d3-ease";
 export * from "d3-format";
+export * from "d3-hierarchy";
```

Comment:
Should not be in common - the only d3 items in common are ones used by common...